### PR TITLE
Add a fixedfooter placement style

### PR DESCRIPTION
**Pull request by davidfischer**
Adds a `fixedfooter` custom placement.

- [Edit: it is now closeable] Should we make it closeable? I think that's probably a good idea here in case it covers some content.
- Only supports text-only ads. I considered an image placement attached to the bottom but it's probably bad on mobile.

## Screenshots
Screenshot from 2022-06-28 19-56-18
Screenshot from 2022-06-28 19-56-31

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -290,10 +290,38 @@ Using our StickyBox style:
...
+
+FixedFooter
+~~~~~~~~~~~
+
+.. versionadded:: 1.6
+
+The "FixedFooter" style is a floating placement is a text-only placement
+attached to the bottom of the screen.
+By floating, it ensures that the ad is always seen
+(and therefore results in billed views that make the publisher money).
```
Comment (ericholscher) [inline docs/index.rst:309]:
```suggestion
The "FixedFooter" style is a floating, text-only placement
attached to the bottom of the screen.
By floating, it ensures that the ad is always seen
(resulting in the highest view rate, generating the most revenue).
```

**Review by ericholscher — APPROVED** [review]
Excited for this! Should be pretty simple to implement & test for folks, and I really like shipping "no CSS" integrations. 

> Should we make it closeable? I think that's probably a good idea here in case it covers some content.

Yes, I'm 💯 on making all our ads closable in general. I think it's the user-friendly thing to do, and definitely should be done on anything sticky.

> Only supports text-only ads. I considered an image placement attached to the bottom but it's probably bad on mobile.

I think that's probably fine? Optimizing for mobile over having a good ad result probably doesn't make sense for most publishers, but I do think a "full-size" ad in a fixed footer is not the right approach. I think an "always-on" stickybox is probably the right implementation there.

**Comment by davidfischer** [comment]
I made the ad closeable.

**Review by armandogomez0 — APPROVED** [review]

**Comment by armandogomez0** [comment]
![176342370-ccd55ffd-cf82-4fc6-a172-9cec0cd9f607](https://user-images.githubusercontent.com/93067022/177049752-983faf8e-6831-4e03-b37d-71c6361082e9.`

**Pull request merged**